### DISCUSSION TITLE
Pagination Increase Number Between ellipsis

### DISCUSSION THREAD

**Discussion by Vinya007** [yajra/laravel-datatables#2545] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/45792827/101589774-1f5c5f80-3a0f-11eb-9574-8bc255eb6a91.png)

I want increase page number between ...  4 5 6  ...   =>    ... 4 5 6 7 8 9 ...

- Operating System
- PHP Version 7.3.12
- Laravel Version 8.7.1
- Laravel-Datatables Version 9.11.0

**Comment by yajra** [comment, marked as answer]
I think what you need is `$.fn.DataTable.ext.pager.numbers_length = 9;`. Taken from https://datatables.net/reference/option/pagingType.

If not, then you would need to create your own [renderer](https://datatables.net/reference/option/renderer) / pagingType.

**Reply by Vinya007** [reply]
Here is my code please suggest,
I want increase number between ellips

I want increase page number between ellips
Now => 1 ... 4 5 6 ... 18,
I Want => => 1 ... 4 5 6 7 8 9 ... 18.
```php
->parameters([
            'order' => [0, 'desc'],
            "pagingType" => "simple_numbers",
            "renderer" => "bootstrap",
            "ext.pager.numbers_length" => 9,
            "columnDefs" => [
                [
                    'targets' => -1
                ]
            ],
        ]);
```

**Reply by yajra** [reply]
I think it's not an option but an api. Not tested but maybe something like below:

```php
<script>$.fn.DataTable.ext.pager.numbers_length = 9;</script>
{{ $dataTable->scripts() }}
```

**Reply by Vinya007** [reply]
It's working as.
<script>$.fn.DataTable.ext.pager.numbers_length = 9;</script>
{{ $dataTable->scripts() }}
